api/api.py

```diff
@@ -1,39 +1,29 @@
-from fastapi import FastAPI
-from pydantic import BaseModel
+import flask
+import flask_cors
```

Comment:
No need for cors stuff if flask is serving the html template that links the js bundle from flask static dir.